data/query.go

```diff
@@ -112,9 +106,11 @@ func handleValidationError(ctx context.Context, err error, description, id strin
 }
 
 // ReviewDataAggregationQueryWithParams ensures that all search parameter values given by the user are reviewed
-func ReviewDataAggregationQueryWithParams(ctx context.Context, cfg *config.Config, urlQuery url.Values) (sp SearchURLParams, validationErrs []core.ErrorItem) {
+func ReviewDataAggregationQueryWithParams(ctx context.Context, cfg *config.Config, urlQuery url.Values, urlPath string) (sp SearchURLParams, validationErrs []core.ErrorItem) {
 	sp.Query = urlQuery.Get("q")
-
+	if urlPath != "" {
+		sp.URIPrefix = urlPath
+	}
```

Comment:
Now that you're using `ReviewPreviousReleasesQueryWithParams` you don't need this change here.

Reply:
thanks - removed now